frontend/src/components/accounts/Login.js

```diff
@@ -0,0 +1,94 @@
+import { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+
+import { useAuth } from "../../contexts/AuthContext";
+
+export default function Login() {
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [loading, setLoading] = useState(false);
```

Comment:
if there are similar places that you have been commented on, it would be good to fix all such places
fix all cases please
![image](https://github.com/mate-academy/node_chat/assets/70009205/065e96b9-0986-4419-af8b-2dfcfde81742)